docs/dapp_prelaunch.md

```diff
@@ -5,7 +5,7 @@ author: Claude Barde
 
 #### You will find below a few considerations that you may take into account when building your Tezos dapp or before publishing it. These pointers help you prevent eventual bugs and improve your users' experience.
 
-- Is your dapp using a reliable and scalable RPC node? Running your own RPC node is often the best choice, especially if you anticipate significant traffic to your dapp. Using a paid hosted node is also a great idea, such as Nautilus from Cryptonomic, or TezTools. You can find [here](http://tezostaquito.io/docs/rpc_nodes#list-of-community-run-nodes) a list of public Tezos nodes.
+- Is your dapp using a reliable and scalable RPC node? Running your own RPC node is often the best choice, especially if you anticipate significant traffic to your dapp. Using a paid hosted node is also a great idea, such as Nautilus from Cryptonomic, or TezTools. You can find [here](http://taquito.io/docs/rpc_nodes#list-of-community-run-nodes) a list of public Tezos nodes.
```

Comment:
This might be a good chance to change the `http` links to `https`

Reply:
Good catch. Updated!